src/lndUtils.ts

```diff
@@ -469,17 +457,15 @@ export const validate = async ({
     }
 
     pushPayment = true
-    destination = params.destination
     username = params.username
+  }
 
-    const preimage = randomBytes(preimageByteLength)
-    id = createHash("sha256").update(preimage).digest().toString("hex")
-    const secret = preimage.toString("hex")
-    messages = [{ type: keySendPreimageType, value: secret }]
+  if (!!params.amount && !!tokens) {
+    const error = `Invoice contains non-zero amount, but amount was also passed separately`
+    // FIXME: create a new error. this is a not a graphl error.
```

Comment:
We already have a `ValidationError` in the `error.ts`. It looks like the wrong one was imported here in this file.